Dockerfile

```diff
@@ -12,16 +12,18 @@ COPY ./assets/tools/graphclust_tools2.yml $GALAXY_ROOT/tools_2.yaml
 COPY ./assets/tools/graphclust_utils.yml $GALAXY_ROOT/tools_3.yaml
 
 RUN install-tools $GALAXY_ROOT/tools.yaml && \
-    /tool_deps/_conda/bin/conda clean --tarballs --yes && \
-    rm /export/galaxy-central/ -rf
+    /tool_deps/_conda/bin/conda clean --tarballs --yes 
+    # && \
+    # rm /export/galaxy-central/ -rf
```

Comment:
why have you removed this?

Reply:
It helped reduce run-time. I kept the last rm, to clean-up once at after all 3.

Reply:
but this now makes your container bigger ... just saying.